pkg/stream/retry_io.go

```diff
@@ -0,0 +1,100 @@
+package stream
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/justtrackio/gosoline/pkg/cfg"
+	"github.com/justtrackio/gosoline/pkg/cloud/aws/sqs"
+	"github.com/justtrackio/gosoline/pkg/log"
+)
+
+type RetryHandlerIO struct {
+	input    AcknowledgeableInput
```

Comment:
We could embed the input, then we don't have to implement all the methods for the input to forward the calls to the input

Reply:
done